ts/Dashboard/Actions/Bindings.ts

```diff
@@ -133,6 +180,9 @@ class Bindings {
             case 'DataGrid': 
                 component = DataGridComponent.fromJSON(json as DataGridComponent.ClassJSON);
                 break;
+            // case 'kpi': 
```

Comment:
Remove?

Reply:
Its intentional to do not miss it. 

`toJSON` method should be added in `KPI component`. I will create a separate ticket in Asana.